rootfs/etc/confd/templates/builder

```diff
@@ -136,11 +139,19 @@ kubectl --namespace=$POD_NAMESPACE create -f /etc/${SLUG_NAME}.yaml
 
 #check for image creation or slug existence in S3EP
 
-while [ ! -f /apps/${SLUG_NAME}/slug.tgz ]
-do
-  sleep 2
-  echo "no file"
-done
+if [[ "$REMOTE_STORAGE" == "1" ]]; then
+  LS_CMD="$MC_PREFIX ls $PUSH_URL"
+  until $LS_CMD &> /dev/null; do
+    echo "no file"
+    sleep 2
+  done
+else
+  while [ ! -f /apps/${SLUG_NAME}/slug.tgz ]
+  do
+    sleep 2
+    echo "no file"
+  done
```

Comment:
@smothiki we can remove it for now, but a progress bar doesn't make much sense because the build is binary from builder's perspective. I thing a better long term solution is #44